dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/net/RequestExecutionContext.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.api.net
+
+/**
+ * Provides information about the current request execution context.
+ * @param attempts the number of attempts made to execute the request in case of a retry.
+ * @param previousResponseCode the response code of the previous request failure code in case of a retry.
```

Comment:
Seems like this sentence should be corrected. It should add at least the comma and state, that this is a code of the previous attempt, otherwise one may think that it can be the code of the previous batch upload.

Reply:
I am going to try to be more explicit here.